### PR TITLE
Update account-map.csv.20220722.verified

### PR DESCRIPTION
**Pull request by manishbafna**
My account info added

### CODE REVIEW

File: migration/mappings-data/account-map.csv.20220722.verified [hunk 1]
```diff
@@ -169,3 +169,4 @@ mharwood,markharwood,Mark Harwood
 hossman,hossman,Chris M. Hostetter
 munendrasn,munendrasn,Munendra S N
 vajda,ovalhub,Andi Vajda
+manish1982,manishbafna,Manish
```
Comment (mocobeta) [inline migration/mappings-data/account-map.csv.20220722.verified:172]: We don't see the username `manish1982` in Lucene Jira, I wonder if you mean `manish82`?
```
manish82,Manish
```

Comment (mocobeta) [reply migration/mappings-data/account-map.csv.20220722.verified:172]:
```suggestion
manish82,manishbafna,Manish
```
I suggest this change - @manishbafna if you are fine with it I'll commit this in.

Comment (manishbafna) [reply migration/mappings-data/account-map.csv.20220722.verified:172]: image
My username in JIRA is manish1982.

Comment (mocobeta) [reply migration/mappings-data/account-map.csv.20220722.verified:172]: Ok, but`manish1982` has no activity in Lucene Jira so far.
https://issues.apache.org/jira/secure/ViewProfile.jspa?name=manish1982

On the other hand, `manish82` has activities in Lucene Jira.
https://issues.apache.org/jira/secure/ViewProfile.jspa?name=manish82

Two Jira users' names are the same - I suspect you have two Jira accounts?

Comment (manishbafna) [reply migration/mappings-data/account-map.csv.20220722.verified:172]: You are right. We can add manish82 to this.

Comment (mocobeta) [reply migration/mappings-data/account-map.csv.20220722.verified:172]: Thanks @manishbafna for confirming. I'll commit this in main soon.

**Comment by mikemccand** [comment]
In general, we could/should just add both Jira accounts (mapping to the same GitHub account), right?  We can dedup the user on import?

In this case it does not matter so much since there is no Jira activity on one of the accounts (and hopefully @manishbafna doesn't suddenly go do a bunch of Jira work with the inactive Jira account in the next few weeks!).

But in general I would just add all the mappings for one user.

**Comment by mocobeta** [comment]
Agreed, `manish1982` also should be added.
https://github.com/apache/lucene-jira-archive/commit/cdd8bceea44913bcdc47495e338916794473881d

**Pull request merged**